capture/run_tasks_pypln.py

```diff
@@ -0,0 +1,28 @@
+from celery import Celery
+import pypln.api
+import pymongo
+import settings
+from tasks_pypln import fetch_property
+from celery import group
+
+
+## Media Cloud database setup
+client = pymongo.MongoClient(host=settings.MONGOHOST)
+articles = client.MCDB.articles # articles collection
+pypln_temp = client.MCDB.pypln_temp # pypln_temp temporary collection
+articles_analysis = client.MCDB.articles_analysis # articles_analysis collection
+
+
+def get_pypln_properties(doc_id):
+    article = pypln_temp.find_one({'_id': doc_id})
+    articles_analysis.insert({'articles_id': article['articles_id']})
+
+    results = fetch_property.delay(doc_id)
+
+    pypln_temp.remove({'_id': doc_id})
```

Comment:
This means that as soon as the `fetch_property` task is started, the document will be removed from the collection. This may be me being too conservative, but I think we may want to only remove this when the work has been done. Maybe do it after saving the analysis itself?